# Opgave I

#### **Spørgsmål I.1 (1)**
Hvilken af nedenstående værdier har den egenskab, at sandsynligheden for, at X1 er lavere end
denne værdi, er 15% (husk at svaret kan være afrundet)?

Tjekker efter med qnorm.

In [108]:
qnorm(c(0.15), mean = 5, sd = 2)

[1] 2.927133

Svar: **3**

#### **Spørgsmål I.2 (2)**
Hvad er sandsynligheden for, at stikprøvegennemsnittet X̄ = 25
P25
i=1 Xi er større end 4.5?

Husker på at det er størren end, så vi siger 1-pnorm(). sd er sigma/sqrt(n) da den følger sætning 3.3 i bogen.

In [109]:
1 - pnorm(4.5, mean=5, sd=2/sqrt(25))

[1] 0.8943502

Svar: **1**

# Opgave II

#### **Spørgsmål II.1 (3)**
I betragtning af at standardafvigelsen for strækning σl og standardafvigelsen for koncentratio-
nen σc er kendt, hvilken af de følgende formler vil approksimere standardafvigelsen af absor-
bansen?

Svar: **4**

#### **Spørgsmål II.2 (4)**
I et eksperiment er den gennemsnitlige strækning bestemt til at være 1 cm med en standardaf-
vigelse på 0.1 cm. Den gennemsnitlige koncentration bestemmes til 0.65 M med en standardaf-
vigelse på 0.09 M. γ er angivet som 0.3 M−1 cm−1 . Hvilke af følgende simuleringer kan bruges
til at bestemme standardafvigelsen for absorbansen?

Enten 1 eller 5, idet 0.09 ikke er opløftet i anden. 5 er korrekt, idet den finder sd.

Svar: **5**

#### **Spørgsmål II.3 (5)**
I ovennævnte spørgsmål blev en tilfældig prøve fra en normalfordeling simuleret ved hjælp af
kommandoen rnorm. Hvilken af kommandoerne nedenfor kan bruges til at simulere en tilfældig
prøve af længde n fra standardnormalfordelingen?

# Opgave III

#### **Spørgsmål III.1 (6)**
Giv et estimat af rateparametrene λ1 og λ2 . Parametrene skal beregnes i kunder pr. time (h−1 ).

Først regner vi mean om fra ventetid i min, til kunder pr time (mean/60min)

In [110]:
c(60/2.76,60/0.897)

[1] 21.73913 66.88963


Voila!

Svar: **4**

#### **Spørgsmål III.2 (7)**
Hvilket af følgende udsagn er korrekt?

Der er blevet udført et parm. boot. 95% CI er fundet til 0.30 3.42. 0 er ikke inde i dette interval, hvorfor middelventetiderne er signifikant forskellige.

Svar: **3**

# Opgave IV

#### **Spørgsmål IV.1 (8)**
Det blev indrapporteret at i alt 92 af harddiskene fejlede indenfor de første tre år af deres
levetid. Angiv den estimerede standardafvigelse, σ̂p̂ , for den estimerede andel af harddiske, som
går i stykker indenfor tre år:

In [111]:
n = (950)
p = (92)/n
# Eller blot
sqrt(p * (1 - p) / n) # e.q (7-6)

[1] 0.009595158

Svar: **3**

#### **Spørgsmål IV.2 (9)**
Virksomheden har en målsætning om, at 90% af deres NAS-harddiske skal have en levetid på
over tre år. Ved hjælp af et statistisk test ønsker de at undersøge, om de lever op til dette mål.
Hvilken statistisk nulhypotese er da relevant at teste?

probabiliteten skal være 10% eller rettere 0.1

Svar: **1**

#### **Spørgsmål IV.3 (10)**
Virksomheden ønsker at undersøge, om de to typer harddiske har samme sandsynlighed for at
fejle indenfor de første tre år af deres levetid. Hvilken af følgende R-koder udfører det relevante
statistiske test?

Vi bruger linje 4, idet det testes for 1 år + 1-2 år + 2-3 år.

Svar: **4**


#### **Spørgsmål IV.4 (11)**
Virksomheden kunne også have valgt at undersøge, om fordelingen af antallet harddiske der
bryder sammen i de fire aldersintervaller er forskellig for de to typer diske. Under den tilsvarende
nulhypotese H0 , hvad er så antallet af almindelige harddiske, som forventes at fejle efter 1-2
år?

Ud fra nulhypotesen antager vi at der er lige stor chance for fejl for begge typer. Det undersøger jeg:

Jeg tager Total væriden for 1-2år for begge grupper. Dividere med den totale mængde respondanter. Ganger til sidst med mængden i den Alm. gruppe:

In [112]:
78/1600 * 650

[1] 31.6875

Svaret bliver 32, hvilket ikke er en mulighed.

Svar: **5**

#### **Spørgsmål IV.5 (12)**
Antag at virksomheden faktisk udfører et χ2 -test for at undersøge, om fordelingen af antallet af
harddiske, der bryder sammen i de fire aldersintervaller, er forskellig for de to typer harddiske.
Hvor mange frihedsgrader har den χ2 -fordeling, som anvendes i hypotesetestet?

Fra bogene finder formlen for df på side 140:

In [113]:
# v = n - 1
4 - 1

[1] 3

Svar: **3**

# Opgave V

#### **Spørgsmål V.1 (13)**
Hvad er variansen af X?

For en Poissonfordeling: mu = var. Med et blackout per uge giver ddet en rate på 1 (da enhenden er pr uge) (sætning 2.28). 

Svar: **3**

#### **Spørgsmål V.2 (14)**
Hvad er sandsynligheden for at der ikke er noget blackout på en tilfældigt udvalgt dag?

Raten var 1 per uge, men vi omregner til per dag, hvilket bliver 1/7

In [114]:
dpois(x=0, lambda=1/7)

[1] 0.8668779

Svar: **5**

# Opgave VI

#### **Spørgsmål VI.1 (15)**
Hvad bliver p-værdien for testet?

Sætning 8.6:

In [115]:
# k = 5, n = 30
1 - pf(4.3, 5-1, 30-5)

[1] 0.008766031

Svar: **1**

# Opgave VII

#### **Spørgsmål VII.1 (16)**
Som det ses, mangler der nogle tal. Ved B burde der stå:

Bruger tabellen fra bogens side 357 til at udlede:

In [116]:
SSTr = 1.19 
SEE = 4.53
MSE = SEE/(9)
MSTr = SSTr / (2)
(Fobs = MSTr/MSE)

[1] 1.182119

Svar: **2**

#### **Spørgsmål VII.2 (17)**
Det oplyses, at der var lige mange observationer i hver gruppe. Hvor mange observationer var
der i én af grupperne?

Fra treatm df = 2, ved vi at der er 3 grupper. Idet Residuals df = 9 = n - k = n - 3 => n = 12.
12 observationer fordelt på 3 grupper, dvs 4 obs. pr gruppe.

Svar: **3**

# Opgave VIII

#### **Spørgsmål VIII.1 (18)**
Ifølge analyserne hvad er da den gennemsnitlige årlige stigning i energiforbrug i perioden esti-
meret til (i ”kg olie ækvalent” pr. år)?

Aflæser Estimate for year som er 21.66. Rundes op til 21.7 og svaret er fundet.

Svar: **4**

#### **Spørgsmål VIII.2 (19)**
Hvad beregnes korrelationen mellem energiforbrug og GDP til?

Bruger formel 5-80.
sd1/sd2 * beta_1

In [117]:
1465/179 * 1.2e-1

[1] 0.9821229

Svar: **3**

#### **Spørgsmål VIII.3 (20)**
Hvilken af følgende konklusioner er mest hensigtsmæssig på baggrund af disse plots (både
konklusion og argument skal være korrekt)?

Fra residualplottet kan vi se at de ikke er fordelit lineært. De kunne godt følge en potens funktion. 

Svar: **4**

#### **Spørgsmål VIII.4 (21)**
Ifølge bogens definition er der da nogle ekstreme observationer i stikprøven, bestående af de ob-
serverede residualer fra den estimerede model mellem energiforbrug og årstal (både konklusion
of argument skal være korrekt)?

Udregner IQR for at kunne afgøre om der ligger data uden of for 1.5 * IQR

In [118]:
IQR = 74.54 - (-60.45)
c(-60.45 - IQR*1.5,74.54 + IQR*1.5)

[1] -262.935  277.025

Har nu fundet grænseværdierne.

Fra plottet ser vi, at ekstrempunkterne er ~160 og ~-130. Disse tal ligger inden for grænserværdierne. 

Svar: **2**

#### **Spørgsmål VIII.5 (22)**
Man udvider nu modellen til en multipel lineær regressionsmodel, ved at anvende både årstallet
og GDP som forklarende variabler.

# Opgave IX

#### **Spørgsmål IX.1 (23)**
Hvad er 95% konfidensintervallet for middelværdien af slagtevægten af grise fra gruppe 1?

In [119]:
109.4 + c(-1,1) * qt(1 - 0.05/2, df=8-1) * 6.2/sqrt(8)

[1] 104.2167 114.5833

eller

In [120]:
x = c(113.3, 117.9, 111.9, 109.6, 109.6, 111.5, 97.8, 103.3)
t.test(x, conf.level = 0.95)


	One Sample t-test

data:  x
t = 49.81, df = 7, p-value = 3.442e-10
alternative hypothesis: true mean is not equal to 0
95 percent confidence interval:
 104.1707 114.5543
sample estimates:
mean of x 
 109.3625 


95% conf intervallet er hermed fundet.

Svar: **1**

#### **Spørgsmål IX.2 (24)**
Man ønsker at bestemme et 99% konfidensinterval for variansen af slagtevægten for gruppe 1.
Hvordan beregnes dette korrekt?

Bruger bogens formel 3-18.
![title](img/m319.png)

Husker på at det er varians vi er efter, derfor 3-18 og IKKE 3-19.

Den eneste linje der er korrekt er mulighed 1, idet 8-1 = 7 og variansen er i anden. 

Svar: **1**

#### **Spørgsmål IX.3 (25)**
Ved test for forskellen i slagtevægt mellem gruppe 1 og gruppe 2, hvad fåes da den sædvanligt
anvendte Welch teststatistik til?

Kunne regne den i hånde via:
![title](img/m349.png)

eller bruge t.test():

In [121]:
a = c(113.3, 117.9, 111.9, 109.6, 109.6, 111.5, 97.8, 103.3)
b = c(110.7, 108.3, 110.6, 106.7, 109.7, 107.5, 105.9, 111.0, 99.9, 110.2, 99.4, 103.6)
t.test(a,b)


	Welch Two Sample t-test

data:  a and b
t = 0.96491, df = 11.028, p-value = 0.3553
alternative hypothesis: true difference in means is not equal to 0
95 percent confidence interval:
 -3.078118  7.886452
sample estimates:
mean of x mean of y 
 109.3625  106.9583 


Svar: **1**

#### **Spørgsmål IX.4 (26)**
Hvis man i et nyt forsøg ønsker en styrke på 80% for at kunne detektere en forskel på 4 kg
mellem de to grupper på konfidensniveau 99%, og bruger den sammenvægtede varians som gæt
på populationens varians, hvor mange grise skal så mindst indgå i dette forsøg?

In [122]:
# pooled variance
s = ((8 - 1)*6.2^2 + (12-1)*4.1^2) / (8 + 12 - 2)
power.t.test(power=0.80, delta=4, sd=sqrt(s), sig.level=0.01, type="two.sample")


     Two-sample t test power calculation 

              n = 38.51164
          delta = 4
             sd = 5.022118
      sig.level = 0.01
          power = 0.8
    alternative = two.sided

NOTE: n is number in *each* group


Vi har to grupper, dvs 2 * 39 = 78.

Svar: **4**

# Opgave X

#### **Spørgsmål X.1 (27)**
Denne analyse vil bedst laves ved:

Man kan sammenligne to steder af gangen, for at afgøre om metode eller lokation har indflydelse. Derfor to-vejs.

Svar: **2**

# Opgave XI

#### **Spørgsmål XI.1 (28)**
Testet for hypotesen af, at middelkvaliteten for de tre maskiner er ens, foretages ved at vurdere
teststørrelsen i en:

Vi har 3 maskiner, med 11 observationer. Fra two-way anova får vi følgende frihedsgrader:
k - 1 og n - k
Hvilket bliver 
2 og 8.

Svar: **4**

#### **Spørgsmål XII.1 (29)**
Hvilke fire korrelationskoefficienter (i rækkefølgen: a), b), c), d)) passer bedst med figuren?

a) har en umiddelbart ens fordeling, derfor lav korrelation (e.g 0.02)

b) har en middel linear korrelation, med upwards trend (e.g 79)

c) har en stor linear korrelation, med downwards trend (e.g -97)

d) har en stor linear korrelation, med upwards trend (e.g. 95)

Svar: **5**

#### **Spørgsmål XII.2 (30)**
Hvilken af følgende er svar er det eneste som ikke er meget usandsynligt?

ved x = 4 skal y = 0.



In [123]:
# 0 = A + 4 * b
# Ved A = 20 og B = -5
20 + 4 * (-5)

[1] 0

Svar: **2**